### PR TITLE
Add doc for DescribeFeatureType and Transaction WFS requests

### PR DESCRIPTION
**Pull request by pblottiere**
This PR needs https://github.com/qgis/QGIS-Documentation/pull/7228 to be merged first (and will need a rebase after that).

### CODE REVIEW

**Comment by pblottiere** [comment]
This PR is rebased and ready to be reviewed.

**Review by DelazJ — COMMENTED** [review]
Could you begin applying some of the changes logic in #7246, here? No need to do a second round.

File: docs/server_manual/services/wfs.rst [hunk 1]
```diff
@@ -182,3 +182,241 @@ URL example:
...
+  &FEATUREID=24
+
+.. note::
+
+  ``FEATUREID``, ``FILTER`` and ``BBOX`` parameters are mutually exclusive.
+
+.. _`qgisserver-wfs-transaction-add`:
+
+Add features
+^^^^^^^^^^^^
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst:728]: I'm not sure, I think we already have too much sub-levels. But I don't have a strong opinion on this matter...

File: docs/server_manual/services/wfs.rst [hunk 2]
```diff
@@ -570,3 +570,241 @@ URL example:
...
+  http://localhost/qgisserver?
+  SERVICE=WFS
+  &VERSION=1.1.0
+  &REQUEST=Transaction
+  &OPERATION=DELETE
+  &FEATUREID=24
+
+.. note::
+
+  ``FEATUREID``, ``FILTER`` and ``BBOX`` parameters are mutually exclusive.
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst]: I added a quick note about that point.

File: docs/server_manual/services/wfs.rst [hunk 3]
```diff
@@ -570,3 +570,241 @@ URL example:
...
+
+
+.. csv-table::
+   :header: "Parameter", "Required", "Description"
+   :widths: auto
+
+   ":ref:`MAP <qgisserver-wfs-map>`", "Yes", "Specify the QGIS project file"
+
+
+.. _`qgisserver-wfs-describefeaturetype-request`:
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst]: Done

File: docs/server_manual/services/wfs.rst [hunk 4]
```diff
@@ -20,8 +20,8 @@ Standard requests provided by QGIS Server:
 
    ":ref:`GetCapabilities <qgisserver-wfs-getcapabilities>`", "Returns XML metadata with information about the server"
    ":ref:`GetFeature <qgisserver-wfs-getfeature>`", "Returns a selection of features"
-   "DescribeFeatureType", "Returns a description of feature types and properties"
-   "Transaction", "Allows features to be inserted, updated or deleted"
+   ":ref:`DescribeFeatureType <qgisserver-wfs-describefeaturetype>`", "Returns a description of feature types and properties"
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst]: Done

**Comment by pblottiere** [comment]
Hi @DelazJ,

I updated this PR according to your comments. Let me know what you think :).

Thanks!

**Review by DelazJ — COMMENTED** [review]
Here we go

File: docs/server_manual/services/basics.rst [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+  Not available for REST based services like :ref:`WFS3 (OGC API Features)
+  <ogc_api_features>`.
+
+
+.. _services_basics_short_name:
+
+Short name
+----------
+
+A number of elements have both a ``<Name>`` and a ``<Title>``.  The **Name** is
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/basics.rst:128]: TBH, I just moved the section from the `Advanced Configuration` chapter... But yes, in this case ``<Name>`` is actually the short name.

I could try to reword this chapter, just let me know.

File: docs/server_manual/services/wcs.rst [hunk 6]
```diff
@@ -33,9 +33,9 @@ WCS 1.1.1 specifications:
    :header: "Parameter", "Required", "Description"
    :widths: auto
 
-   ":ref:`SERVICE <wcs_service>`", "Yes", "Name of the service (**WCS**)"
+   ":ref:`SERVICE <services_basics_service>`", "Yes", "Name of the service (**WCS**)"
+   ":ref:`REQUEST <services_basics_request>`", "Yes", "Name of the request (**GetCapabilities**)"
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wcs.rst:37]: Actually the version number is rarely used in QGIS Server implementation, so the ``REQUEST`` parameter looks like much more discriminating to me (at least when reading/writing code). But I don't have a strong opinion on the matter so we can keep the current order if you prefer.

File: docs/server_manual/services/wcs.rst [hunk 7]
```diff
@@ -81,24 +81,7 @@ service:
...
-.. _`wcs_getcapabilities_request`:
-
-REQUEST
-^^^^^^^
-
-This parameter is ``GetCapabilities`` in case of the **GetCapabilities**
-request.
-
-
-.. _`wcs_version`:
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wcs.rst:101]: Oh ok, I don't have this kind of "concerns" in vim :sweat_smile:.

File: docs/server_manual/services/wfs.rst [hunk 8]
```diff
@@ -570,3 +512,224 @@ URL example:
...
+
+.. csv-table::
+   :header: "Parameter", "Required", "Description"
+   :widths: auto
+
+   ":ref:`SERVICE <services_basics_service>`", "Yes", "Name of the service (**WFS**)"
+   ":ref:`REQUEST <services_basics_request>`", "Yes", "Name of the request (**DescribeFeatureType**)"
+   ":ref:`VERSION <wfs_version>`", "No", "Version of the service"
+   ":ref:`OUTPUTFORMAT <wfs_getfeature_outputformat>`", "No", "Format of the response"
+   ":ref:`TYPENAME <wfs_describefeaturetype_typename>`", "No", "Name of layer"
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst]: Fixed

File: docs/server_manual/services/wfs.rst [hunk 9]
```diff
@@ -570,3 +512,224 @@ URL example:
...
+extra parameters:
+
+.. csv-table::
+   :header: "Parameter", "Required", "Description"
+   :widths: auto
+
+   ":ref:`MAP <services_basics_map>`", "Yes", "QGIS project file"
+
+
+.. _wfs_describefeaturetype_typename:
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wfs.rst]: Yes indeed, it's fixed.

File: docs/server_manual/services/wms.rst [hunk 10]
```diff
@@ -1817,43 +1744,54 @@ This is useful to ensure scale based visibility of layers and
...
 * show if a group in the layer tree is :ref:`mutually exclusive <group_layers_interact>`
 
+The **GetProjectSettings** request supports the following parameters:
+
+.. csv-table::
+   :header: "Parameter", "Required", "Description"
+   :widths: auto
 
-.. _server_wms_getschemaextension:
+   ":ref:`MAP <services_basics_map>`", "Yes", "QGIS project file"
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wms.rst:1789]: The `MAP` vendor parameter is the first thing checked in QGIS Server whatever the underlying service/request. So I think it makes sense to keep it in first position. But again, I don't have a strong opinion on the matter so let me know what you prefer.

File: docs/training_manual/qgis_server/wms.rst [hunk 11]
```diff
@@ -200,9 +200,9 @@ to render only **Germany** and **Italy** from the countries layer.
 |moderate| |FA| Use Redlining
 -------------------------------------------------------------------------------
 
-Let's do another GetMap request that makes use of the :ref:`qgisserver-redlining`
+Let's do another GetMap request that makes use of the :ref:`wms_redlining`
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/training_manual/qgis_server/wms.rst]: Done

File: docs/server_manual/services/wms.rst [hunk 12]
```diff
@@ -486,7 +433,7 @@ Opacity can be set on layer or group level. Allowed values range from
 0 (fully transparent) to 255 (fully opaque).
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wms.rst]: Done

File: docs/server_manual/services/wms.rst [hunk 13]
```diff
@@ -852,47 +792,39 @@ and item labels:
...
-
-This parameter is ``GetLegendGraphics`` in case of the **GetLegendGraphics** request.
-
-
-.. _`wms-getlegendgraphics-bbox`:
+   ":ref:`MAP <services_basics_map>`", "Yes", "QGIS project file"
+   ":ref:`SRCWIDTH <wms_getlegendgraphics_srcwidth>`", "No", "Map width"
+   ":ref:`SRCHEIGHT <wms_getlegendgraphics_srcheight>`", "No", "Map height"
+   ":ref:`SHOWFEATURECOUNT <wms_getlegendgraphics_showfeaturecount>`", "No", "Add feature count of features"
+   ":ref:`RULE <wms_getlegendgraphics_rule>`", "No", ""
```
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/wms.rst]: Oops, done

**Comment by pblottiere** [comment]
Hello @DelazJ. Thanks a lot for your review, let me know what you think with the last modifications :).

File: docs/server_manual/services/basics.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/server_manual/services/basics.rst:129]: Yes, please. Replacing name with short name should be enough, I think

File: docs/server_manual/services/wcs.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/server_manual/services/wcs.rst:37]: No strong opinion, either! So I rely on your more advanced experience with QGIS Server users' needs.

File: docs/server_manual/services/wcs.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/server_manual/services/wcs.rst:101]: :smiling_face_with_tear:  I use vim only when there's no other option. kate is simply no-brainer.

File: docs/server_manual/services/wms.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/server_manual/services/wms.rst:1808]: same as above

File: docs/training_manual/qgis_server/wms.rst [hunk 14]
```diff
@@ -1,4 +1,4 @@
-|LS| Serving WMS
+RRc| Serving WMS
```
Comment (DelazJ) [inline docs/training_manual/qgis_server/wms.rst:1]:
```suggestion
|LS| Serving WMS
```

**Comment by DelazJ** [comment]
Found a typo and other than the short name description, lgtm

File: docs/server_manual/services/basics.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (pblottiere) [reply docs/server_manual/services/basics.rst]: Done

**Comment by pblottiere** [comment]
> Found a typo and other than the short name description, lgtm

Typo is fixed and I updated the short name chapter. Ready to be merged? :)

**Pull request merged**